docusaurus/docs/Android/04-compose/07-guides/01-adding-custom-attachments.mdx

```diff
@@ -308,15 +308,17 @@ class MessagesActivity : AppCompatActivity() {
 
 Date attachments should be now correctly rendered in the message composer and in the message list like on the screenshot below:
 
-![Rendering Custom Attachment](../../assets/compose_rendering_custom_attachments.png)
+|![Rendering Custom Attachment](../../assets/compose_rendering_custom_attachments.png)|
+|---|
 
 With just a few lines of code, you're able to render any custom UI within the `MessageComposer` and `MessageList` components of our SDK. This gives the ability to build powerful features without having to change our default components — you just need a custom `AttachmentFactory`!
 
 ## Quoted Messages
 
 Stream SDK supports quoting or replying to messages, even if they contain attachments. These quoted messages are shown inside the message bubble above the text you wrote when quoting, which is a common pattern in various chat services.
 
-![Compose Quoted Attachment Sample](../../assets/compose_quoted_attachment_example.PNG)
+|![Compose Quoted Attachment Sample](../../assets/compose_quoted_attachment_example.png)|
```

Comment:
I think we should update this seeing how we're now updating our samples with stock photos instead of faces of team mates.